ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -124,6 +124,11 @@ public static final class Headers {
      * has been modified after the value set for this header.
      */
     public static final String IF_MODIFIED_SINCE = "If-Modified-Since";
+    /**
+     * Header to be set by clients during a Put blob call to clearly specify that the client is aware of the
+     * UPDATE feature of named blob, and client wants to update when the named blob already exist.
+     */
+    public static final String PUT_UPDATE = "x-ambry-update";
```

Comment:
I think it would help to be more specific.  There's already HTTP PUT verb.  As a user, I wouldn't know when to set this header and what value to set to.  Is this the name of the named blob?  If so, consider "x-ambry-blob-name"?

Reply:
Updated, so by default the update for named-blob will be upsert. User can specify a header 'x-ambry-named-not-upsert' to disable this default upsert (see the latest commit).